### PR TITLE
DIP-228 Remove Graph Change Announcement

### PR DESCRIPTION
**Pull request by wesbiggs**
Problem
=======
Graph Change Announcement is too limited and will be replaced with mutable state management operations. See DIP #228.

Solution
========
This PR removes references to Graph Change Announcement. The new solution will be addressed in a separate PR.

Change summary:
---------------
- [ ] Removed Graph Change Announcement page
- [ ] Removed link from navigation section
- [ ] Removed from Announcement Type table
- [ ] Added to pending spec changes
- [ ] Removed a reference in Frequency docs that was now a broken link. (Note, however, that this PR does not attempt to update Frequency documentation.)

Steps to Verify:
----------------
1. Review content changes.
2. Generate site (`npm run build`) and check changed pages.

### CODE REVIEW

File: pages/Frequency/Publishing.md [hunk 1]
```diff
@@ -1,14 +1,13 @@
 # Announcement Publishing
 
 On Frequency, [Announcements](../DSNP/Announcements.md) are mapped to Schemas which in turn publish Frequency Messages.
-Frequency Messages are either direct Graph Changes from a particular user, or a Batch Publication with a multitude of possible users.
+Frequency Messages are either mutable state changes from a particular user, or a Batch Publication with a multitude of possible users.
```
Comment (shannonwells) [inline pages/Frequency/Publishing.md:4]: This implies an implementation cannot be DSNP compliant with if users can post their own announcements one at a time. This seems overspecified. What is the justification for preventing this (as opposed to simply supporting/allowing announcements in batches)?

Comment (wesbiggs) [reply pages/Frequency/Publishing.md:4]: This page is for the Frequency spec, not DSNP. I just tweaked it so it didn't refer to Graph Change Announcements anymore. But this whole part of the spec site needs to be overhauled and updated outside the scope of this DIP.

**Review by shannonwells — APPROVED** [review]

File: pages/Frequency/Publishing.md (hunk 1 above)
Comment (wilwade) [inline pages/Frequency/Publishing.md:4]: Messages are never mutable state changes, but I get what is trying to happen here.

Perhaps:

```suggestion
On Frequency, [Announcements](../DSNP/Announcements.md) are mapped to Schemas which in turn publish Frequency Messages or state changes.
Frequency Messages can be from a particular user, or a Batch Publication with a multitude of possible users.
```

File: pages/Frequency/Publishing.md [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 
 <!-- Links to https://libertydsnp.github.io/frequency should be updated with links to docs.frequency.xyz when able to be -->
 
 | Announcement Type Enum | Announcement | Batched | Schema Id Mainnet | Schema Id Rococo | Frequency Model Type | Frequency Payload Location |
 | --- | --- | --- | --- | --- | --- | --- |
 | 0 | [Tombstone](../DSNP/Types/Tombstone.md) | YES | TBD | 1 (_v1.1.0+_) | [`Parquet`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.ModelType.html#variant.Parquet) | [`IPFS`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.IPFS) |
-| 1 | [Graph Change](../DSNP/Types/GraphChange.md) | no | TBD | 7 (_v1.1.0+_) | [`AvroBinary`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.ModelType.html#variant.AvroBinary) | [`OnChain`](https://libertydsnp.github.io/frequency/common_primitives/schema/enum.PayloadLocation.html#variant.OnChain) |
```
Comment (wilwade) [inline pages/Frequency/Publishing.md:11]: Hmm... I know we likely will not publish Graph Change to mainnet Frequency, but feels odd just removing it.

Feels like it should have a version of `<= v1.1.0` or something. If we were to change another in the future, the older value (may) still matter to read historical data.

**Review by wilwade — CHANGES_REQUESTED** [review]
Questions and one minor change to make for clarity.

**Review by wilwade — APPROVED** [review]
👍

**Pull request merged**